docs/database.rst

```diff
@@ -0,0 +1,31 @@
+database.py module
+==================
+
+It includes various database operations like add, delete, update and display the bookmarks.
+
+Following are the methods available in this module under class DatabaseConnection.
+
+Methods
+=======
+
+1. __init__
+2. init_db
+3. add_url
+4. tag_url
+5. delete_url
+6. update_url
+7. show_url
+8. search_by_tag
+9. delete_all_url
+10. check_url_db
+11. open_url
+
+System modules
+======================
+
+1. BeautifulTable
+2. datetime
+3. webbrowser
+4. sqlite3
+5. os
```

Comment:
@daivshala, Please add the small description of the system modules w. r. t. our project.